server/repository/odp/markAsActual.ts

```diff
@@ -2,7 +2,7 @@ import { insertAudit } from '@server/repository/audit/auditRepository'
 import { OdpClassRepository } from '@server/repository'
 import { getAndCheckOdpCountryId } from './getAndCheckOdpCountryId'
 
-export const markAsActual = async (client: any, odpId: any, user: any) => {
+export const markAsActual = async (client: any, odpId: any, user: any): Promise<null | [void, void]> => {
```

Comment:
Not sure about this? Thoughts?

Reply:
what about Promise<void> only?
and instead of returning null, you can return Promise.resolve() ?

Reply:
Can't return only promise: `TS2314: Generic type 'Promise ' requires 1 type argument(s).`

Reply:
Resolved in next pr